startup/00-startup.py

```diff
@@ -42,3 +44,34 @@
 # import logging
 # ophyd.logger.setLevel(logging.DEBUG)
 # logging.basicConfig(level=logging.DEBUG)
+
+# Enabling (2018C3) logging of the message hook into a log file.
+import os
+import sys
+import datetime
+import functools
+
+# TODO: to be removed and imported from bluesky once
+# https://github.com/NSLS-II/bluesky/pull/1117 is reviewed, merged, released,
+# and pushed to the beamline.
+def ts_msg_hook(msg, file=sys.stdout):
+    t = '{:%H:%M:%S.%f}'.format(datetime.datetime.now())
+    msg_fmt = '{: <17s} -> {!s: <15s} args: {}, kwargs: {}'.format(
+        msg.command,
+        msg.obj.name if hasattr(msg.obj, 'name') else msg.obj,
+        msg.args,
+        msg.kwargs)
+    print('{} {}'.format(t, msg_fmt), file=file)
```

Comment:
@danielballan, should we consider [`flush=True`](https://docs.python.org/3/library/functions.html#print) here as well?